components/google_sheets/actions/update-row/update-row.mjs

```
@@ -62,32 +71,63 @@ export default {
-        throw new ConfigurationError("Could not find a header row. Please either add headers and click \"Refresh fields\" or adjust the action configuration to continue.");
-      }
-      const { values: rowValues } = !isNaN(row)
-        ? await this.googleSheets.getSpreadsheetValues(sheetId, `${worksheet?.properties?.title}!${row}:${row}`)
-        : {};
-      for (let i = 0; i < values[0]?.length; i++) {
-        props[`col_${i.toString().padStart(4, "0")}`] = {
+        if (!values?.[0]?.length) {
+          throw new ConfigurationError("Could not find a header row. Please either add headers and click \"Refresh fields\" or set 'Does the first row of the sheet have headers?' to false.");
+        }
+
+        const { values: rowValues } = !isNaN(row)
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)